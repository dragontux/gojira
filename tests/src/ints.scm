; ints
(display 0)
(newline)

(display 123)
(newline)

(display -1)
(newline)

(display 268435455)
(newline)

(display (+ 1 2 3 4))
(newline)

(display (* 2 (- 123 321)))
(newline)

(display (/ 100 2))
(newline)
