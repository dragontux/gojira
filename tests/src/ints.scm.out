0
123
-1
268435455
10
-396
50
